src/core/glib-utils.c

```diff
@@ -418,7 +418,7 @@ _g_strdup_with_max_size (const char *s,
 		char *first_half;
 		char *second_half;
 		int   offset;
-		int   half_max_size = max_size / 2 + 1;
+        int   half_max_size = max_size / 2 + 1;n_fields
```

Comment:
Although this typo is commented out, please reverse it.

Reply:
Done